src/components/select/select.spec.js

```diff
@@ -1010,6 +1013,14 @@ describe('<md-select>', function() {
         expect(ngModelCtrl.$valid).toBe(true);
       });
 
+      it('should have the proper md-has-input-class when the model has selected values', function() {
+        $rootScope.model = [2,4,5,6];
+        var el = setupSelectMultiple('ng-model="$root.model"', [1,2,3,4,5,6]);
+
+        expect(el).toHaveClass('md-input-has-value');
+
```

Comment:
Minor nit ;) this blank line can be removed.

Reply:
Done 🐔